tests/deposits.test.ts

```diff
@@ -2,6 +2,35 @@ import { expect } from 'chai'
 import { Mappings } from './wasm'
 
 describe('onDeposit', function () {
+  it('creates a deposit entity', async () => {
+    const mappings = await Mappings.load()
+
+    const txHash = `0x${'42'.repeat(32)}`
+    mappings.onDeposit(
+      {
+        user: '0x0000000000000000000000000000000000000001',
+        token: '0x0000000000000000000000000000000000000002',
+        amount: 10n ** 18n,
```

Comment:
What is the `n` for?

Reply:
For `BigInt`. NodeJS has native `BigInt` support and appending an `n` to a number literal creates it as a `BigInt` instead of a `Number`.